external/criu/pingPerf.sh

```diff
@@ -86,7 +86,8 @@ prepare() {
         curCommitID=$(git rev-parse HEAD)
         echo "Using dockerfile from OpenLiberty/ci.docker repo branch $openLibertyBranch with commit hash $curCommitID"
         libertyDockerfilePath="releases/latest/beta/Dockerfile.ubi.openjdk17"
-        sed -i 's;FROM icr.io\/appcafe\/ibm-semeru-runtimes:open-17-ea-jdk-ubi-amd64;FROM local-ibm-semeru-runtimes:latest;' $libertyDockerfilePath
+        # replace OpenLiberty dockerfile base image
+        sed -i 's;FROM icr.io\/appcafe\/ibm-semeru-runtimes:open-17-jdk-ubi;FROM local-ibm-semeru-runtimes:latest;' $libertyDockerfilePath
```

Comment:
In case the OpenLiberty changes this again, we can add a check? If the string that we are looking for does not exist, fail the build? For example:

```
imageFrom="FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-jdk-ubi;"
        grep "${imageFrom}" $libertyDockerfilePath
        rtCode=$?
        if [ $rtCode == 0 ]; then
            echo "Replace ${imageFrom} to FROM local-ibm-semeru-runtimes:latest;"
            sed -i 's;FROM icr.io\/appcafe\/ibm-semeru-runtimes:open-17-jdk-ubi;FROM local-ibm-semeru-runtimes:latest;' $libertyDockerfilePath
        else
            echo "Cannot find ${imageFrom} in https://github.com/OpenLiberty/ci.docker/blob/instanton/releases/latest/beta/Dockerfile.ubi.openjdk17"
            exit 1
        fi
```

Reply:
The above code will not work as we have space in imageFrom. Maybe we can use `q` command in sed:
https://stackoverflow.com/questions/15965073/return-value-of-sed-for-no-match/15965681#15965681

Reply:
We have decided to do this in a separate PR so the weekly run will fail.